### ISSUE TITLE
Logs

### ISSUE THREAD

**Issue by hirenshah** [charlyie/resmushit-wordpress#7]
I've posted on the WP and checked everywhere possible but cannot seem to find where the logs get written to. Is there somewhere obvious that I've completely missed?

Just in case something weird had happened, I've uninstalled and reinstalled the plugin, as well as toggling the logging option. 

I have everything running via cron and I know it's working because the images are being changed.

**Comment by hirenshah** [comment]
I ended up going through your code and have worked out what is wrong.  In _resmushit.inc.php_ the following returns _FALSE_ if the file does not exist.

```
if( !is_writable(ABSPATH . RESMUSHIT_LOGS_PATH) ) {
	return FALSE;
}

```
As soon as I went and created it, it immediately started writing to it :)

That check should actually be:
```
if( !is_writable(ABSPATH) ) {
	// Do Something Here to Inform User
	return FALSE;
}
```

Then extend the code that follows it with an _else_, and cater for the scenario of a file not existing:

```
if(file_exists(ABSPATH . RESMUSHIT_LOGS_PATH)){
	if(filesize(ABSPATH . RESMUSHIT_LOGS_PATH) > RESMUSHIT_LOGS_MAX_FILESIZE) {
		$logtailed = logtail(ABSPATH . RESMUSHIT_LOGS_PATH, 20);
		$fp = fopen(ABSPATH . RESMUSHIT_LOGS_PATH, 'w');
		fwrite($fp, $logtailed);
		fclose($fp);
	}
} else {
	// Create New File
	// Do Whatever Else
}
```

**Comment by charlyie** [comment]
Hi @hirenshah,

Thanks for your feedback, it has been fixed in the last release (0.3.1) !

Cheers,
Charles
